lib/solargraph/position.rb

```diff
@@ -82,9 +88,15 @@ def self.from_offset text, offset
       cursor = 0
       line = 0
       character = nil
-      text.lines.each do |l|
+      text.each_line do |l|
         line_length = l.length
-        char_length = l.chomp.length
+
+        if l.end_with?("\n") || l.end_with?("\r\n")
```

Comment:
```suggestion
        if l.end_with?("\n")
```

If something ends with `\r\n` it also ends with `\n` making the `|| l.end_with?("\r\n")` useless